### PR TITLE
Dewey to additionalClassificationDdc

### PR DESCRIPTION
**Pull request by kwahlin**
See https://jira.kb.se/browse/FMT-202

### CODE REVIEW

File: librisworks/src/main/groovy/se/kb/libris/mergeworks/compare/Classification.groovy [hunk 1]
```diff
@@ -49,18 +53,12 @@ class Classification extends StuffSet {
...
     }
 
-    boolean isSab(Map c) {
-        c['inScheme'] && c['inScheme']['code'] =~ 'kssb'
-    }
-
-    String maxSabVersion(c1, c2) {
-        def v1 = c1['inScheme']['version'] ?: "-1"
-        def v2 = c2['inScheme']['version'] ?: "-1"
-        Integer.parseInt(v1) > Integer.parseInt(v2) ? v1 : v2
```
Comment (olovy) [inline librisworks/src/main/groovy/se/kb/libris/mergeworks/compare/Classification.groovy:59]: Should these really be deleted?

File: librisworks/src/main/groovy/se/kb/libris/mergeworks/compare/Classification.groovy [hunk 2]
```diff
@@ -69,10 +67,59 @@ class Classification extends StuffSet {
...
+            def allDewey = instanceDocs.collect { it.classification() }
+                    .flatten()
+                    .findAll { Map c -> isDewey(c) }
+
+            def preferredDewey = findPreferredDewey(deweyOnMerged, allDewey)
+
+            deweyOnMerged.remove(preferredDewey)
+
+            mergedWork['classification'] = asList(mergedWork['classification']) - deweyOnMerged
+            mergedWork['additionalClassificationDdc'] = (asList(mergedWork['additionalClassificationDdc']) + deweyOnMerged).unique()
```
Comment (olovy) [inline librisworks/src/main/groovy/se/kb/libris/mergeworks/compare/Classification.groovy:86]: This is a bit hard to follow. I suggest calling `deweyOnMerged` something else here?

**Review by olovy — APPROVED** [review]

File: librisworks/src/main/groovy/se/kb/libris/mergeworks/compare/Classification.groovy (hunk 2 above)
Comment (kwahlin) [reply librisworks/src/main/groovy/se/kb/libris/mergeworks/compare/Classification.groovy]: See [ea7368e](https://github.com/libris/librisxl/pull/1327/commits/ea7368e0fa7613e3c2e7b4acf555e0d1e317df35)
